### ISSUE TITLE
Sites asked me to turn off Adblocker

### ISSUE THREAD

**Issue by TungBox** [beakerbrowser/beaker#1056]
Operation System: Mac OS X
Beaker version:  Beaker v0.8.0-prerelease.6
Hi,
Some sites are asking me to turn off Adblocker while im browsing it on Beaker, Is there anyway to turn the adblocker off?
Thanks

**Comment by taravancil** [comment]
Sorry, we don’t currently have a way to turn the built-in adblocker off.

There is an issue open for adding an icon to tabs that are playing audio,
but sadly it’s not high on our todo list.

**Comment by pfrazee** [comment]
I'm going to turn this into an enhancement issue.

**Comment by jraspiprojects** [comment]
Wait there's an adblocker on beaker? I never knew because ghostery was always installed on my firefox browser.

**Comment by pfrazee** [comment]
Yeah, just a simple URL blacklist. It may not make it into 0.8.0 final because of an issue with upstream though

**Comment by pfrazee** [comment]
Note: We've had to temporarily disable the adblocker due to an issue with the module. We'll re-enable when that's solved.
